dev-db/sqlite/README.md

```diff
@@ -0,0 +1,11 @@
+This is a fork of gentoo's `dev-db/sqlite` package. The main
+reasons for having our fork are:
+
+1. Even in case of sqlite >= 3.32.0, we still need to keep conditions to
+   distinguish full archive from non-full archive. Main reason for that is
+   because it is not possible to build sqlite from its full archive (i.e.
+   "-src"), which depends `dev-lang/tcl`. Either we need to make
+   `dev-lang/tcl` available in the Flatcar SDK, or we need to bring back
+   the ability of dealing with its non-full archive. (i.e. "-autoconf")
+   Obviously the latter is much easier, so we bring back the non-full
```

Comment:
Why is it easier to maintain a fork than to include dev-lang/tcl in the SDK? This seems counter-intuitive to me. I'd have expected the other one to be easier.

Reply:
You are right. Let me rephrase:
```
While the former is ideal way to go in the future, it will require additional updates in the SDK-only release.
The latter is easier for us to resolve the on-going security issues in the short term.
```

Reply:
Sure, yes, this makes a lot more sense :)

Reply:
Updated the README.